### ISSUE TITLE
Failed to check if peer is already present - loop

### ISSUE THREAD

**Issue by karek314** [LiskHQ/lisk-sdk#3415]
Randomly, node stops syncing with network and enters infinite loop. After fully stopping Lisk core and starting up again, it resolves itself and catch up with network.
```
WRN] 2019-04-14 00:28:02 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 145.239.134.21:8001
[inf] 2019-04-14 00:28:03 | Releasing enqueued broadcasts
[inf] 2019-04-14 00:28:03 | Queue empty
[WRN] 2019-04-14 00:28:03 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 54.92.3.168:8001
[WRN] 2019-04-14 00:28:03 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 116.25.44.76:42021
[WRN] 2019-04-14 00:28:04 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 35.200.58.219:8001
[WRN] 2019-04-14 00:28:04 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 148.251.182.170:8001
[WRN] 2019-04-14 00:28:04 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 47.52.68.17:8001
[WRN] 2019-04-14 00:28:05 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 213.222.210.80:8001
[WRN] 2019-04-14 00:28:05 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 13.230.103.143:8001
[WRN] 2019-04-14 00:28:05 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 13.114.212.150:8001
[WRN] 2019-04-14 00:28:06 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 52.196.55.71:8001
[WRN] 2019-04-14 00:28:06 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 35.195.11.25:8001
[WRN] 2019-04-14 00:28:06 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 192.121.166.208:8001
[WRN] 2019-04-14 00:28:07 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 13.209.44.12:8001
[WRN] 2019-04-14 00:28:08 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 192.99.5.85:8001
[inf] 2019-04-14 00:28:08 | Releasing enqueued broadcasts
[inf] 2019-04-14 00:28:08 | Queue empty
[WRN] 2019-04-14 00:28:08 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 94.177.222.65:8001
[WRN] 2019-04-14 00:28:09 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 178.128.243.61:8001
[WRN] 2019-04-14 00:28:09 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 85.203.20.5:8001
[WRN] 2019-04-14 00:28:10 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 18.185.110.148:8001
[WRN] 2019-04-14 00:28:10 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 45.77.55.54:8001
[WRN] 2019-04-14 00:28:10 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 213.136.88.142:8001
[WRN] 2019-04-14 00:28:10 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 13.230.14.157:8001
[WRN] 2019-04-14 00:28:10 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 94.130.90.159:8001
[WRN] 2019-04-14 00:28:11 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 54.249.145.198:8001
[WRN] 2019-04-14 00:28:12 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 47.90.29.31:5000
[WRN] 2019-04-14 00:28:12 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 13.114.212.150:8001
[WRN] 2019-04-14 00:28:13 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 52.194.38.246:8001
[WRN] 2019-04-14 00:28:13 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 13.230.132.123:8001
[inf] 2019-04-14 00:28:13 | Releasing enqueued broadcasts
[inf] 2019-04-14 00:28:13 | Queue empty
[WRN] 2019-04-14 00:28:13 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 13.52.53.187:8001
[WRN] 2019-04-14 00:28:14 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 209.222.98.250:8001
[WRN] 2019-04-14 00:28:14 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 220.194.43.238:8001
[WRN] 2019-04-14 00:28:15 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 35.171.245.24:8001
[WRN] 2019-04-14 00:28:15 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 120.92.11.175:8001
[WRN] 2019-04-14 00:28:16 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 52.194.169.85:8001
[WRN] 2019-04-14 00:28:17 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 54.250.195.159:8001
[WRN] 2019-04-14 00:28:17 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 23.108.213.121:8001
[WRN] 2019-04-14 00:28:17 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 148.251.54.6:8001
[WRN] 2019-04-14 00:28:18 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 47.96.115.21:8001
[inf] 2019-04-14 00:28:18 | Releasing enqueued broadcasts
[inf] 2019-04-14 00:28:18 | Queue empty
[WRN] 2019-04-14 00:28:19 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 5.135.141.51:8001
[WRN] 2019-04-14 00:28:20 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 23.108.213.121:8001
[WRN] 2019-04-14 00:28:20 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 47.92.254.255:8001
[WRN] 2019-04-14 00:28:20 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 220.194.43.238:8001
[WRN] 2019-04-14 00:28:21 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 52.9.25.223:8001
[WRN] 2019-04-14 00:28:22 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 93.115.28.167:8001
[WRN] 2019-04-14 00:28:22 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 83.23.41.169:8001
[WRN] 2019-04-14 00:28:22 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 112.74.56.7:8001
[WRN] 2019-04-14 00:28:22 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 150.140.159.220:8001
[WRN] 2019-04-14 00:28:22 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 52.194.38.246:8001
[WRN] 2019-04-14 00:28:23 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 3.17.154.11:8001
[WRN] 2019-04-14 00:28:23 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 34.220.106.199:8001
[WRN] 2019-04-14 00:28:23 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 165.227.169.93:8001
[WRN] 2019-04-14 00:28:23 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 94.130.131.212:8001
[inf] 2019-04-14 00:28:23 | Releasing enqueued broadcasts
[inf] 2019-04-14 00:28:23 | Queue empty
[WRN] 2019-04-14 00:28:24 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 88.99.235.240:8001
[WRN] 2019-04-14 00:28:24 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 88.99.95.216:8001
[WRN] 2019-04-14 00:28:24 | Peer INSERT error: code: 4200, message: Failed to check if peer is already present, description: Procedure list not registered on WAMPServer. Available commands: [object Object] on peer 94.130
```

### Steps to reproduce
Perhaps running blockchain snapshot script in background. Node has also forging enabled and multiple API endpoints are used extensively. Hardware is regular 7200 RPM drive with 128gb Ram.

### Which version(s) does this affect? (Environment, OS, etc...)
1.5.1 but It's out there for much longer.

**Comment by karek314** [comment]
Some additional logs
`[dbg] 2019-04-22 17:26:11 | Updated peer 220.178.235.85:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | Updated peer 80.211.223.199:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | Updated peer 165.227.20.141:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | Updated peer 104.237.2.167:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | Updated peer 110.143.155.171:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | Updated peer 185.33.145.247:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | Updated peer 163.172.166.145:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | Updated peer 80.211.154.39:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | Updated peer 80.211.151.215:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | Updated peer 80.211.131.233:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | Updated peer 80.211.245.218:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | [Outbound socket :: close] Peer connection to 52.194.169.85:8001 closed with code 4007 and reason - undefined
[dbg] 2019-04-22 17:26:11 | [Outbound socket :: error] Peer error from 52.194.169.85:8001 - Client connection establishment timed out
[log] 2019-04-22 17:26:11 | GET /api/node/status from 127.0.0.1
[dbg] 2019-04-22 17:26:11 | Listing peers - {"attempt":"matched broadhash","found":440,"matched":0,"picked":0,"accepted":0}
[dbg] 2019-04-22 17:26:11 | Listing peers - {"attempt":"unmatched broadhash","found":440,"matched":287,"picked":100,"accepted":100}
[dbg] 2019-04-22 17:26:11 | Network height is: 8939984
[dbg] 2019-04-22 17:26:11 | Cache - Get value for key: transactionCount | Status: true
[dbg] 2019-04-22 17:26:11 | Updated peer 35.205.133.91:8001 - {"state":2,"broadhash":"53aac2b3c36c67c623c9e640137ea2bda75915fff9736c57e64b5cef91379b1a","height":8939985}
[dbg] 2019-04-22 17:26:11 | Updated peer 195.181.218.50:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | Updated peer 31.14.134.83:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | Updated peer 80.211.202.72:8001 - {"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:11 | [Outbound socket :: close] Peer connection to 112.74.56.7:8001 closed with code 4007 and reason - undefined
[dbg] 2019-04-22 17:26:11 | [Outbound socket :: error] Peer error from 112.74.56.7:8001 - Client connection establishment timed out
[log] 2019-04-22 17:26:11 | GET /api/node/transactions/unprocessed?limit=100&offset=0 from 127.0.0.1
[dbg] 2019-04-22 17:26:11 | [Outbound socket :: close] Peer connection to 52.194.38.246:8001 closed with code 4007 and reason - undefined
[dbg] 2019-04-22 17:26:11 | [Outbound socket :: error] Peer error from 52.194.38.246:8001 - Client connection establishment timed out
[dbg] 2019-04-22 17:26:12 | [Outbound socket :: close] Peer connection to 94.130.131.212:8001 closed with code 4007 and reason - undefined
[dbg] 2019-04-22 17:26:12 | [Outbound socket :: error] Peer error from 94.130.131.212:8001 - Client connection establishment timed out
[dbg] 2019-04-22 17:26:12 | Broadhash consensus: 0 %
[inf] 2019-04-22 17:26:12 | Releasing enqueued broadcasts
[inf] 2019-04-22 17:26:12 | Queue empty
[log] 2019-04-22 17:26:12 | GET /api/node/transactions/unprocessed?limit=100&offset=0 from 127.0.0.1
[dbg] 2019-04-22 17:26:13 | Updated peer 83.23.111.27:8001 - {"state":2}
[dbg] 2019-04-22 17:26:13 | [Outbound socket :: close] Peer connection to 35.205.133.91:8001 closed with code 4007 and reason - undefined
[dbg] 2019-04-22 17:26:13 | [Outbound socket :: error] Peer error from 35.205.133.91:8001 - Client connection establishment timed out
[dbg] 2019-04-22 17:26:13 | Updated peer 5.135.141.51:8001 - {"state":2,"broadhash":"53aac2b3c36c67c623c9e640137ea2bda75915fff9736c57e64b5cef91379b1a","height":8939985}
[dbg] 2019-04-22 17:26:13 | Updated peer 45.77.55.54:8001 - {"state":2,"broadhash":"53aac2b3c36c67c623c9e640137ea2bda75915fff9736c57e64b5cef91379b1a","height":8939985}
[dbg] 2019-04-22 17:26:13 | Updated peer 213.136.88.142:8001 - {"state":2,"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[log] 2019-04-22 17:26:13 | GET /api/node/transactions/unprocessed?limit=100&offset=0 from 127.0.0.1
[dbg] 2019-04-22 17:26:14 | Updated peer 94.130.69.23:8001 - {"state":2,"broadhash":"91111eaffb590dd2004b174ab2595de1c360f3b893985f3426e55b74b62dade6","height":8939986}
[dbg] 2019-04-22 17:26:14 | Updated peer 52.9.25.223:8001 - {"state":2}
[dbg] 2019-04-22 17:26:14 | Updated peer 95.216.227.100:8001 - {"state":2}
[log] 2019-04-22 17:26:14 | GET /api/node/status from 127.0.0.1
[dbg] 2019-04-22 17:26:14 | Listing peers - {"attempt":"matched broadhash","found":440,"matched":0,"picked":0,"accepted":0}
[dbg] 2019-04-22 17:26:14 | Listing peers - {"attempt":"unmatched broadhash","found":440,"matched":291,"picked":100,"accepted":100}`

**Comment by ManuGowda** [comment]
This issue can be closed after `2.0.0` release, as it will be fixed as part of #3417
